modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -37,8 +64,58 @@ The Deploy Application page opens:
+
+. Verify that https://www.mulesoft.com/platform/saas/cloudhub-ipaas-cloud-based-integration[CloudHub access] is enabled on Anypoint Platform.
+. If you do not already have access to the Anypoint-CLI command line tool, install it using xref:runtime-manager::anypoint-platform-cli.adoc#installation[Anypoint Platform CLI installation instructions].
+. Log into your Anypoint Platform account from the command line:
+  .. Enter your username: `anypoint-cli --username="user"`. 
+  .. Enter your password.
+. Use the `runtime-mgr application deploy` command providing the *name* of the app and the *location* of the deployable archive (`.zip`) file on your file system, for example:
++
+----
+runtime-mgr cloudhub-application deploy myMuleApp /Users/exported-app-folder/my-mule-app.zip
+----
++
+. If there are no issues with the name, location or any optional parameters provided, you should see a table like this:
++
+----
+Deploying myMuleApp ...
+┌──────────────────────────────┬────────────────────────────────────────┐
+│ Domain                       │ mymuleapp.cloudhub.io                  │
+├──────────────────────────────┼────────────────────────────────────────┤
+│ Status                       │ UNDEPLOYED                             │
+├──────────────────────────────┼────────────────────────────────────────┤
+│ Updated                      │ a few seconds ago                      │
+├──────────────────────────────┼────────────────────────────────────────┤
+│ Runtime                      │ 4.1.0                                  │
+├──────────────────────────────┼────────────────────────────────────────┤
+│ File name                    │ my-mule-app.zip                        │
+├──────────────────────────────┼────────────────────────────────────────┤
+│ Persistent queues            │ false                                  │
+├──────────────────────────────┼────────────────────────────────────────┤
+│ Persistent queues encrypted  │ false                                  │
+├──────────────────────────────┼────────────────────────────────────────┤
+│ Static IPs enabled           │ false                                  │
+├──────────────────────────────┼────────────────────────────────────────┤
+│ Monitoring                   │ Enabled. Auto-restart if not responding│
+├──────────────────────────────┼────────────────────────────────────────┤
+│ Workers                      │ 1 vCore * 1                            │
+----
+
++
+. Congratulations, your app is now deployed on CloudHub!
 
-=== From SandBox
++
+[TIP]
+. Applications that are deployed via the CLI can be viewed and managed on Runtime Manager like any other application.
+  .. If you then want to stop the app, on the command line type `runtime-mgr cloudhub-application stop <app-name>`.
+  .. To exit the command line tool, hit `ctrl + c` twice.
+. xref:runtime-manager::anypoint-platform-cli.adoc[Anypoint Platform CLI] contains a full list of commands.
```

Comment:
Same comment as above. Should be:
xref:anypoint-platform-cli2.adoc

Reply:
fixed & tested locally